server/models/Collective.js

```diff
@@ -1454,13 +1455,15 @@ export default function(Sequelize, DataTypes) {
             ]),
           },
         };
-        promises.push(
-          models.Activity.create({
-            CollectiveId: this.id,
-            type: activities.COLLECTIVE_APPLY,
-            data,
-          }),
-        );
+        if (options.activity && options.activity !== activities.COLLECTIVE_CREATED_GITHUB) {
```

Comment:
This is wrong. This will opt out everything that doesn't have an activity.

Reply:
Oh didn't think it through, thanks.